core/execution/common/liquidity_provision_fees.go

```diff
@@ -84,6 +85,8 @@ func (m *MarketLiquidity) AllocateFees(ctx context.Context) error {
 	scoresPerLp := m.liquidityEngine.GetAverageLiquidityScores()
 	// Multiplies each equity like share with corresponding score.
 	updatedShares := m.updateSharesWithLiquidityScores(sharesPerLp, scoresPerLp)
+	// now adjust the score to account for the AMM share
+	updatedShares = m.mergeScores(updatedShares, ammScores)
```

Comment:
From my reading of the spec, this new network parameter isn't strictly related to AMM's. 

Say the network parameter is 0.5, it means half the fees are distrubuted calcuated with the current way by doing `ELS * LP-score`, and the other half are distributed using `LP-score` only. Theres no mention of amm's in that part of the spec.

Reply:
Indeed, this has changed a bit, but we are going to have to calculate the LP scores for the AMMs, and add them to the map (which I'm now doing).